x-pack/plugins/fleet/server/services/fleet_server/saved_object_migrations.ts

```diff
@@ -166,8 +167,10 @@ async function migrateAgentPolicies() {
     perPage: SO_SEARCH_LIMIT,
   });
 
+  let hasAgents = false;
   await Promise.all(
     agentPolicies.map(async (agentPolicy) => {
+      hasAgents = true;
```

Comment:
why are we setting this flag on migrating agent policies? shouldn't we set it in `migrateAgents`?

Reply:
Yes totally my bad will fix that